Extensions/Strict.php

```diff
@@ -0,0 +1,85 @@
+<?php
+namespace BulkGate\Magesms\Extensions;
+
+/**
+ * Class Strict
+ * @package BulkGate\Magesms\Extensions
+ */
+class Strict
+{
+    /**
+     * @param $name
+     * @return mixed
+     * @throws Exceptions\StrictException
+     */
+    public function __get($name)
+    {
+        $class = get_class($this);
+
+        if (property_exists($class, $name)) {
+            return $this->$name;
+        }
+
+        throw new Exceptions\StrictException("Cannot read an undeclared property $class::\$$name.");
+    }
+
+    /**
+     * @param $name
+     * @param $value
+     * @throws Exceptions\StrictException
+     */
+    public function __set($name, $value)
+    {
+        $class = get_class($this);
+
+        if (property_exists($class, $name)) {
+            $this->$name = $value;
```

Comment:
Tady by měl být return vidím, ale tam bude problém i u mě. Tady to eskaluje na Exception vždy.

Reply:
https://github.com/BulkGate/extensions/commit/8352425db814db833cfa3c241f5d29e65d53d3b2

Reply:
Diky